README.md

```diff
@@ -277,6 +281,22 @@ perfume.endPaint('togglePopover');
 
 ![Performance](https://github.com/Zizzamia/perfume.js/blob/master/docs/src/assets/performance-cfp.png)
 
+Or you can try the [emerging](https://chromestatus.com/features#elementtiming) [Element Timing API](https://wicg.github.io/element-timing/) specification by simply adding the `elementtiming` attribute to HTML elements you would like to measure:
```

Comment:
We should probably have his own section like
```markup
### Element Timing
brief description

how to use it

demo result
```

Reply:
Done in [https://github.com/Zizzamia/perfume.js/pull/124/commits/dd7d7acfbe2cfd8da4dc4c391fe38b22d36b895e ](https://github.com/Zizzamia/perfume.js/pull/124/commits/dd7d7acfbe2cfd8da4dc4c391fe38b22d36b895e)